x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/value_badge.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import classNames from 'classnames';
+
+import './value_badge.scss';
+
+export const ValueBadge: React.FC<{ children: React.ReactNode; disabled?: boolean }> = ({
```

Comment:
you shouldn't need to specify `children`, it's inherited from React.FC

```suggestion
export const ValueBadge: React.FC<{ disabled?: boolean }> = ({
```

Reply:
I guess the only difference is that by adding it, it makes `children` required. I think having the TS hint that children is required for this component is helpful.

Reply:
Ahh cool! No worries then, ++ for leaving it in